dotCMS/src/main/java/com/dotcms/util/TimeMachineUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.dotcms.util;
+
+import com.dotcms.api.web.HttpServletRequestThreadLocal;
+
+import javax.servlet.http.HttpSession;
+import java.util.Optional;
+
+public class TimeMachineUtil {
```

Comment:
Codacy found an issue: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5835963)

Reply:
Doc